app/styles/modules/_global_header.scss

```diff
@@ -0,0 +1,39 @@
+header {
+  background: #4d4d4d;
```

Comment:
@johngruen is the official Sass owner, but I'll let you guys decide if you want to go with inline values versus declaring "allowed" colors in the <https://github.com/mozilla/chronicle/blob/master/app/styles/_variables.scss> file.